docs/framework/network-programming/how-to-configure-network-tracing.md

```diff
@@ -12,84 +12,85 @@ helpviewer_keywords:
+|`System.Net.Sockets`|Some public methods of the <xref:System.Net.Sockets.Socket>, <xref:System.Net.Sockets.TcpListener>, <xref:System.Net.Sockets.TcpClient>, and <xref:System.Net.Dns> classes.|  
+|`System.Net`|Some public methods of the <xref:System.Net.HttpWebRequest>, <xref:System.Net.HttpWebResponse>, <xref:System.Net.FtpWebRequest>, and <xref:System.Net.FtpWebResponse> classes, and SSL debug information (invalid certificates, missing issuers list, and client certificate errors).|  
 |`System.Net.HttpListener`|Some public methods of the <xref:System.Net.HttpListener>, <xref:System.Net.HttpListenerRequest>, and <xref:System.Net.HttpListenerResponse> classes.|  
 |`System.Net.Cache`|Some private and internal methods in `System.Net.Cache`.|  
 |`System.Net.Http`|Some public methods of the  <xref:System.Net.Http.HttpClient>,  <xref:System.Net.Http.DelegatingHandler>,  <xref:System.Net.Http.HttpClientHandler>, <xref:System.Net.Http.HttpMessageHandler>,  <xref:System.Net.Http.MessageProcessingHandler>, and  <xref:System.Net.Http.WebRequestHandler> classes.|  
 |`System.Net.WebSockets.WebSocket`|Some public methods of the <xref:System.Net.WebSockets.ClientWebSocket> and <xref:System.Net.WebSockets.WebSocket> classes.|  
   
- The attributes listed in the following table configure trace output.  
+ The attributes listed in the following table configure trace output:
   
 |Attribute name|Attribute value|  
 |--------------------|---------------------|  
-|`Value`|Required <xref:System.String> attribute. Sets the verbosity of the output. Legitimate values are `Critical`, `Error`, `Verbose`, `Warning`, and `Information`.<br /><br /> This attribute must be set on the \<add name> element of the \<switches> element as shown in the example. An exception is thrown if this attribute is set on the \<source> element.|  
-|`maxdatasize`|Optional <xref:System.Int32> attribute. Sets the maximum number of bytes of network data included in each line trace. The default value is 1024.<br /><br /> This attribute must be set on the \<source> element as shown in the example. An exception is thrown if this attribute is set on an element under the \<switches> element.|  
-|`Tracemode`|Optional <xref:System.String> attribute. Set to `includehex` to show protocol traces in hexadecimal and text format. Set to `protocolonly` to show only text. The default value is `includehex`.<br /><br /> This attribute must be set on the \<switches> element as shown in the example. An exception is thrown if this attribute is set on an element under the \<source> element.|  
+|`Value`|Required <xref:System.String> attribute. Sets the verbosity of the output. Legitimate values are `Critical`, `Error`, `Verbose`, `Warning`, and `Information`.<br /><br />This attribute must be set on the \<add name> element of the \<switches> element. An exception is thrown if this attribute is set on the \<source> element.|  
```

Comment:
I have a suspicion this should also be lowercase as shown in the example. You could keep the sentence as shown in the example since it's there.

```suggestion
|`value`|Required <xref:System.String> attribute. Sets the verbosity of the output. Legitimate values are `Critical`, `Error`, `Verbose`, `Warning`, and `Information`.<br /><br />This attribute must be set on the \<add name> element of the \<switches> element. An exception is thrown if this attribute is set on the \<source> element.|  
```

Reply:
Attribute names (not values) in XML for configuration are case-insensitive. So, practically speaking, it doesn't matter.

However, lowercase is preferred. It is actually what appears in the Visual Studio Intellisense.  See 'name' attribute below:
![image](https://user-images.githubusercontent.com/7684835/67317139-855dee80-f4be-11e9-9aea-e70b512a0a20.png)

So, I would prefer if the documention used lowercase for all the descriptions of the attributes. That would match what Visual Studio Intellisense is showing.